ChapmanUniversity1.0/Controllers/CoursesController.cs

```diff
@@ -1,36 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.InteropServices;
-using System.Runtime.InteropServices.ComTypes;
-using System.Threading.Tasks;
+﻿using System.Linq;
+using ChapmanUniversity1._0.DAL;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Mvc.Rendering;
-using Microsoft.EntityFrameworkCore;
-using ChapmanUniversity1._0.Data;
 using ChapmanUniversity1._0.Models;
-using ChapmanUniversity1._0.Repositories;
-using Microsoft.AspNetCore.Razor.Language.Intermediate;
+
 
 namespace ChapmanUniversity1._0.Controllers
 {
 
     public class CoursesController : Controller
     {
-        private readonly UnitOfWork.UnitOfWork _unitOfWork = new(new SchoolContext());
+        private readonly UnitOfWork _unitOfWork = new();
```

Comment:
Initialize this variable in a `CoursesController` constructor instead of just newing it up.